lean-circuit/LeanCircuit/SemanticEquivalence.lean

```diff
@@ -10,8 +10,11 @@ open Semaphore (F Order)
 
 variable [Fact (Nat.Prime Order)]
 
+-- Number of levels in Merkle Tree
 abbrev D := 20
 
+-- Start of misc lemmas
```

Comment:
this is highly insuficient – these are the lemmas that actually need some explanation